liquid.py

```diff
@@ -237,17 +260,60 @@ def alloc(job, group):
     print(first(running, 'running allocations'))
 
 
-def nomad_address():
+def collection_exists(name):
+    """Returns true if the collection with the given name was declared in the ini file."""
+
+    names = dict(config.collections).keys()
+    return reduce(lambda exists, curr_name: exists or curr_name.lower() == name.lower(), names, False)
```

Comment:
Collection name can contain upper case letters, unlike the ES index name. This is a requirement. So when comparing collection names I must compare them as case insensitive.

The problem with allowing the user to write directly to the config file is that he/she can write an invalid name and there's no way to stop him/her. My solution would be to generate an index name that is valid from the collection name, but I'd do this in a separate PR.

Reply:
Why is uppercase a requirement? If we're to display user-friendly collection titles, let's make it a separate field, e.g. `title`.

Reply:
I don't see the need for a separate field. The collection name is the title and it may contain upper case letters - this was a requirement from Stephan.

Reply:
The collection needs a clean unique machine-readable ID that should avoid any confusion about names of elasticsearch indices and query parameters for the UI. Let's set it explicitly instead of divining it by some undocumented transformation from an arbitrary unicode string.

Reply:
Sorry, I don't understand what you mean by "transformation from an arbitrary unicode string"

AFAIK, there's no confusion regarding the names of ES indices and query parameters for the UI.

For the moment, we should keep the same rules we had on docker-setup for the collection name. Later we can change that if needed.

Reply:
Ok, I understand your concern, we should be able to migrate existing collections painlessly to this new system. So let's do it the right way, and use the [title](https://github.com/hoover/snoop2/blob/master/snoop/data/views.py#L13) field of snoop's API for the title, as it was intended, but we were too lazy to use a different environment variable.

Reply:
I made an issue (https://github.com/hoover/snoop2/issues/148) to allow the title to be set via a separate environment variable, so then it becomes easy to define collection titles in `liquid.ini`:

```ini
[collection:testdata]
title = TestData Anÿŧĥĩŋģ ☃️📯
```

Reply:
I suppose you're referring to the collection template. But in what part of that template would you inject the title. Anyway, for this small enhancement I'd rather do a separate PR and not keep adding requirements to this one, in order to keep the PRs small.

Reply:
Inject the title as env var to snoop api and workers. Ok to make it a separate PR but please drop the `.lower()` for now, so we have clear sane collection IDs.

Reply:
But I can't drop `.lower()`: this would allow conflicting ES index names for different collections. E.g.: collection `Testdata1` -> index `testdata1`, collection `testdata1` -> index `testdata1`. The same index name for 2 different collections.

Reply:
OK, forced all collection names to contain only lowercase letters.